{"name": "Account creation", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 4s, while waiting for:\nbrowser.element(('css selector', '#email')).has attribute 'value' with value 'mailfortesting13@mailfortestingonly.com'\nReason: AssertionError: actual attribute_value: mailfortesting48@mailfortestingonly.com\nScreenshot: file:///root/.selene/screenshots/1581075049417/1581075049418.png\nPageSource: file:///root/.selene/screenshots/1581075049417/1581075049418.html\n\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.8/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/steps.py\", line 175, in step_impl\n    s(\"#email\").should(have.value(REGISTRATION_EMAIL))\n  File \"/usr/local/lib/python3.8/site-packages/selene/core/entity.py\", line 294, in should\n    super().should(condition)\n  File \"/usr/local/lib/python3.8/site-packages/selene/core/entity.py\", line 121, in should\n    self.wait.for_(condition)\n  File \"/usr/local/lib/python3.8/site-packages/selene/core/wait.py\", line 100, in for_\n    raise self._hook_failure(failure)\n"}, "steps": [{"name": "Given I am in Sign In page", "status": "passed", "start": 1581075050567, "stop": 1581075054037}, {"name": "When I enter mailfortesting48@mailfortestingonly.com in Create New Account section", "status": "passed", "start": 1581075054037, "stop": 1581075056277}, {"name": "And I enter valid account details", "status": "passed", "start": 1581075056277, "stop": 1581075057545}, {"name": "And click on Register button", "status": "passed", "start": 1581075057545, "stop": 1581075060046}, {"name": "And My Account page is opened", "status": "passed", "start": 1581075060046, "stop": 1581075060112}, {"name": "And I click on My Personal Information button", "status": "passed", "start": 1581075060112, "stop": 1581075060936}, {"name": "Then Your Personal Information page is opened", "status": "passed", "start": 1581075060936, "stop": 1581075060971}, {"name": "And correct personal information is displayed", "status": "failed", "statusDetails": {"message": "selene.core.exceptions.TimeoutException: Message: \n\nTimed out after 4s, while waiting for:\nbrowser.element(('css selector', '#email')).has attribute 'value' with value 'mailfortesting13@mailfortestingonly.com'\nReason: AssertionError: actual attribute_value: mailfortesting48@mailfortestingonly.com\nScreenshot: file:///root/.selene/screenshots/1581075049417/1581075049418.png\nPageSource: file:///root/.selene/screenshots/1581075049417/1581075049418.html\n\n", "trace": "  File \"/usr/local/lib/python3.8/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/usr/local/lib/python3.8/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/steps.py\", line 175, in step_impl\n    s(\"#email\").should(have.value(REGISTRATION_EMAIL))\n  File \"/usr/local/lib/python3.8/site-packages/selene/core/entity.py\", line 294, in should\n    super().should(condition)\n  File \"/usr/local/lib/python3.8/site-packages/selene/core/entity.py\", line 121, in should\n    self.wait.for_(condition)\n  File \"/usr/local/lib/python3.8/site-packages/selene/core/wait.py\", line 100, in for_\n    raise self._hook_failure(failure)\n"}, "start": 1581075060972, "stop": 1581075065124}], "start": 1581075050567, "stop": 1581075065124, "uuid": "ac7f3548-c0cf-41fb-95c7-574cb4937ab4", "historyId": "65deff046e9e25a740783ffb86da98a7", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "automationpractice"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}